modules/oe_webtools_analytics/modules/oe_webtools_analytics_rules/README.md

```diff
@@ -0,0 +1,100 @@
+--------------
+
+A rule consists of the following data:
+
+### Site section
+
+The site section that will be included in the analytics request. For more
+details see [Site
+sections](https://webgate.ec.europa.eu/CITnet/confluence/display/NEXTEUROPA/Adapting+the+Next+Europa+Piwik+Module+%3A+add+sections).
+
+### Regular expression
+
+The regular expression that is used to match the page URI. These are standard
+PCRE expressions. For example to match all pages that have a URI that starts
+with `/news/` you can use the regular expression `|^/news/|`.
+
+### Match translated pages on the path alias for the default language
+
+An option to perform the matching on the path alias for the translation in the
+default language of the website. This is intended to facilitate the creation of
+rules for websites that have a policy to publish all content in the default
+language, and translate the content into different languages.
+
+For example, if your default language is English and your news articles have
+paths that start with `/news/` then you can enable this option and provide
+`|^/news/.+|` as the regular expression. The rule would then also be applied to
+the translated news articles with paths `/fr/nouvelles/` and `/nl/nieuws/`.
+
+
+Known issues and limitations
+----------------------------
+
+* By design this module relies only on the URI (a.k.a. "path") of a page to
+  determine the site section. Since the URI is dynamic and does not follow
+  strict rules this cannot be 100% relied upon; a content editor may decide to
+  create a unique path for a page, multiple aliases might exist for the same
+  page, etc.
+
+  However from the perspective of an analytics engineer / SEO expert it is a
+  common practice to rely upon the URI since they do not have intimate knowledge
+  of the site structure and a hierarchical URI is the best possible way they
+  have to determine the content structure. To ensure good results it is highly
+  recommended to use modules such as
+  [Pathauto](https://www.drupal.org/project/pathauto) to provide structured URIs
+  for site content.
+
+  Websites that have more complex requirements are suggested to not use this
+  module but implement a customized subscriber for the `AnalyticsEvent` provided
+  by the Webtools Analytics module.
+* The rules currently have no priority. The rules are checked in the order which
+  they are returned by the database and the first match that is found will
+  determine the site section. This may cause problems if a URI matches multiple
+  rules.
+
+  Ref.
+  [OPENEUROPA-1633](https://webgate.ec.europa.eu/CITnet/jira/browse/OPENEUROPA-1633)
+* The option to match multilingual content on the path alias for the default
+  language only works if a translation exists in the default content of the
+  website.
+
+  Ref.
+  [OPENEUROPA-1637](https://webgate.ec.europa.eu/CITnet/jira/browse/OPENEUROPA-1637)
+* There currently is no support for multilingual content based on system paths.
```

Comment:
I cannot reproduce this issue. `\Drupal\Core\Path\AliasManagerInterface::getAliasByPath()` documentation says that `An alias that represents the path, or path if no alias was found.`.
I tried with a rule for `/admin/people` and it worked for the unaliased default language and for the aliased translated one. Same for a node created in non-default language.

Reply:
Interesting, I also see that there are no test cases referencing this problem. Possibly it has been accidentally solved as part of this issue. I will double check and see if we have coverage for this use case.

Reply:
OK I managed to get it working too. Strangely enough this can be made to work when the option to "match on the default language alias" is *enabled*.  It will then not match on the alias but on the system path :confused:. My test case had this option *disabled* since we're not trying to match on the alias but on the system path. The ticket to fix it proposes to add a third option "Match on system path".

I will update the test case and then the ticket can be closed. I wonder how to explain this case to the end user in the documentation though.

Reply:
OK never mind, it still breaks then after an English language alias is added for the same path. So the bug is still there: it's not possible to reliably create a site section based on the system path - either it doesn't work when there is an alias in the default language, or it doesn't work when there is _no_ alias :confounded:. I'll update the ticket and the README.

Reply:
I have updated the Jira ticket and the information in the "Known issues" section to better explain the problem.